src/asm/rgbasm.5

```diff
@@ -752,12 +752,12 @@ or
 .Ic FRAGMENT
 modifiers, as described below.
 .Ss Unionized Sections
-When you're tight on RAM, you may want to define overlapping blocks of variables, as explained in the
+When you're tight on RAM, you may want to define overlapping memory allocations, as explained in the
```

Comment:
I'm less sure about that one.

Reply:
I based this on other lines like "`DS` allocates a number of empty bytes" and "Unions allow multiple memory allocations to overlap".